pages/login.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: connor
+ * Date: 18/07/2016
+ * Time: 02:12
+ */
+
+define('BASE_URL', 'http://vagrant.learn.jquery.com/wp-admin/admin.php?page=jquery-q-and-a');
```

Comment:
I'm not convinced by this statement. Everything here could change: the vagrant. part and the name of plugin.